src/mmda/predictors/sklearn_predictors/svm_word_predictor.py

```diff
@@ -475,7 +526,12 @@ def _create_words(
         new_word_id = 0
         for token_id in range(1, len(token_id_to_word_id)):
             token = document.tokens[token_id]
-            word_id = token_id_to_word_id[token_id]
+            word_id = token_id_to_word_id.get(token_id)
+            if word_id is None:
```

Comment:
not sure if this warning makes the logs too noisy; was thinking it's useful to know which PDFs need to be re-driven in future when we fix tokenizer

Reply:
Hmm, shouldn't this loop be `for token_id in range(1,len(document.tokens))` to go through all tokens including those that might not have a `word_id`?

Reply:
I'd change this to `logger.debug()`. This will indeed be quite noisy.